addons/mail/models/mail_mail.py

```diff
@@ -40,9 +41,16 @@ def default_get(self, fields):
 
     # content
     mail_message_id = fields.Many2one('mail.message', 'Message', required=True, ondelete='cascade', index=True, auto_join=True)
+
+    # Mail Message fields
```

Comment:
Probably unnecessary comment by the way ^^ field name is clear enough to be put just after mail_message_id